### ISSUE TITLE
Adding escape character

### ISSUE THREAD

**Issue by ajarocki** [enketo/enketo-transformer#36]
Right now there seems to be no way to get strings like "A_B_C" with markdown enabled. Can some sort of escape character be added (typically "\\") so that escaping markdown would be possible, to allow following markdownToHtml transformations:
```
"A_B_C" -> "A<em>B</em>C"
"A\_B\_C" -> "A_B_C"
"_A\_B\_C_" -> "<em>A_B_C</em>"

```
I'm aware that simply making backslash an escape character right now wouldn't be backward-compatible, so to address that there could be some additional flag added (similar to markdown: false).

**Comment by MartijnR** [comment]
I would be much in favor of this favor and welcome a PR (with comprehensive tests)! This is not something on my own current todo list.

Do you think we need that extra setting? If we only look for `\*, \_, \#`, as a start, would that meet your requirements? In that case, it could be activated with `markdown: true` and strip the escape characters. I doubt we'll have backwards compatibility issues for these 3 tags. It gets more complex if we allow escaping everything, including list items etc, but maybe we should start small.

**Comment by MartijnR** [comment]
closed by 7661258e29ca20300ba673e63b1f605256ac746b
